clean.json

```diff
@@ -4,6 +4,7 @@
 		"Pre": [
 			"rm -rf $ARCH/lib/*.a $ARCH/bin/*",
 			"rm -f cfg/pxe/tftpboot/harvey.32bit",
+			"rm -f cfg/pxe/tftpboot/uroot.cpio.lzma",
```

Comment:
quick question, should the cfg here be /cfg?
and same for the rest so this could work no matter where it's run from?

Reply:
Changed - could you re-review please?